src/Types/Node.php

```diff
@@ -22,30 +23,22 @@
  * @psalm-import-type OGMTypes from \Laudis\Neo4j\Formatter\OGMFormatter
  *
  * @psalm-immutable
- *
  * @psalm-immutable
  *
  * @extends AbstractPropertyObject<OGMTypes, int|string|CypherMap<OGMTypes>>
  * @extends AbstractPropertyObject<OGMTypes, int|CypherList<string>|CypherMap<OGMTypes>>
  */
 final class Node extends AbstractPropertyObject
 {
-    private int $id;
-    /** @var CypherList<string> */
-    private CypherList $labels;
-    /** @var CypherMap<OGMTypes> */
-    private CypherMap $properties;
-
     /**
      * @param CypherList<string>  $labels
      * @param CypherMap<OGMTypes> $properties
      */
-    public function __construct(int $id, CypherList $labels, CypherMap $properties)
-    {
-        $this->id = $id;
-        $this->labels = $labels;
-        $this->properties = $properties;
-    }
+    public function __construct(
```

Comment:
Do you know they added new field into structures Node, Relationship and Path? It's element_id. I see you don't have it there yet. https://www.neo4j.com/docs/bolt/current/bolt/structure-semantics/#structure-node

I don't know if it's intentional, I'll just point it out.

Reply:
Good thinking! This is a very early draft but I'll make sure it is there